### PR TITLE
[hi] Add content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md

### PR DESCRIPTION
**Pull request by kranurag7**
Added the localisation of optional-kubectl-configs-bash-mac.md 

(https://github.com/kubernetes/website/issues/30570)

@anubha-v-ardhan

### CODE REVIEW

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+  
+
+| शीर्षक | विवरण | headless |
+| :---         |     :---:      |          ---: |
+| macOS पर बैश ऑटोकम्पलीशन करें   | macOS पर बैश ऑटोकम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन।     | true    |
+
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:6]:
```suggestion
---
title: "macOS पर bash ऑटो-कम्पलीशन"
description: "macOS पर bash ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन।"
headless: true
---
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### परिचय
+`kubectl completion bash` से बैश के लिए kubectl कम्पलीशन स्क्रिप्ट किया जा सकता हैं | इस स्क्रिप्ट को अपने सेल में सोर्स करने से kubectl कम्पलीशन इनेबल होगा |
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:9]:
```suggestion
### परिचय

Bash के लिए kubectl समापन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में समापन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्पलीशन सक्षम हो जाती है।```

Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:9]:
```suggestion
### परिचय

Bash के लिए kubectl समापन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है। आपके शेल में समापन स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्पलीशन सक्षम हो जाती है।```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+हालाँकि, kubectl कम्पलीशन स्क्रिप्ट बैश कम्पलीशन पर निर्भर करती है जिसे आपको पहले इनस्टॉल करना होगा।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:11]:
```suggestion
हालाँकि, समापन की स्क्रिप्ट [**bash-completion**](https://github.com/scop/bash-completion) पर निर्भर हैं जिसका अर्थ है कि आपको पहले इस सॉफ़्टवेयर को इंस्टॉल करना होगा।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+{{चेतावनी>}} बैश कम्पलीशन के लिये दो संस्करण हैं  v1 और v2. v1 bash 3.2 के लिये हैं (जो की macOS के लिए निर्धारित हैं ), और v2 bash 4.1 + के लिए हैं |   
+Kubectl कम्पलीशन स्क्रिप्ट  बैश-कम्पलीशन v1 और बैश 3.2 के साथ ठीक से काम नहीं करती है। इसके लिए बैश-कम्पलीशन v2 और बैश 4.1+ की आवश्यकता है।
+इसलिए macOS पर kubectl कम्पलीशन को सही तरीके से इस्तेमाल करने के लिए , आपको bash इनस्टॉल और उपयोग करना होगा | निम्नलिखित  निर्देश मानते हैं कि आप बैश का उपयोग करते हैं (अर्थात 4.1  का कोई भी बैश संस्करण या इससे नया)
+{{चेतावनी>}}
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:16]:
```suggestion
{{< warning >}}
Bash-completion के लिये दो संस्करण हैं  v1 और v2। v1 bash 3.2 के लिये हैं (जो macOS के लिए डिफ़ॉल्ट है), और v2 bash 4.1+ के लिए है।kubectl कम्पलीशन स्क्रिप्ट Bash-completion v1 और Bash 3.2 के साथ ठीक से **काम नहीं करती है**। इसके लिए **Bash-completion v2** और बैश 4.1+ की आवश्यकता है। इसलिए macOS पर kubectl कम्पलीशन को सही तरीके से इस्तेमाल करने के लिए , आपको bash 4.1+ इनस्टॉल और उपयोग करना होगा ([*निर्देश*](https://itnext.io/upgrading-bash-on-macos-7138bd1066ba))। निम्नलिखित निर्देश मानते हैं कि आप बैश का उपयोग करते हैं (अर्थात 4.1  का कोई भी बैश संस्करण या इससे नया)।
{{< /warning >}}
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+### अपग्रेड बैश 
+
+निम्नलिखित  निर्देश मानते हैं कि आप बैश 4.1+ का उपयोग करते हैं | आप अपने बैश के संस्करण को   यह चलाकर देख सकते हैं:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:20]:
```suggestion
निम्नलिखित निर्देश मानते हैं कि आप बैश 4.1+ का उपयोग करते हैं। आप अपने बैश के संस्करण को यह चलाकर देख सकते हैं:

```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+```bash
+echo $BASH_VERSION
+```
+
+यदि यह बहुत पुराना है, तो आप Homebrew का उपयोग करके इसे इनस्टॉल/अपग्रेड कर सकते हैं :
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:25]:
```suggestion
यदि यह बहुत पुराना है, तो आप Homebrew का उपयोग करके इसे इनस्टॉल/अपग्रेड कर सकते हैं:

```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+```bash
+brew install bash
+```
+
+अपना शेल को दुबारा  लोड करें और सत्यापित करें कि वांछित संस्करण का उपयोग किया जा रहा है:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:30]:
```suggestion
अपने शेल को पुनः लोड करें और सत्यापित करें कि इच्छित संस्करण का उपयोग किया जा रहा है:

```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+```bash
+echo $BASH_VERSION $SHELL
+```
+
+### इनस्टॉल बैश कम्पलीशन
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:35]:
```suggestion
### इनस्टॉल bash-completion
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+यदि यह बहुत पुराना है, तो आप Homebrew का उपयोग करके इसे इनस्टॉल/अपग्रेड कर सकते हैं :
+```bash
+brew install bash
+```
+
+अपना शेल को दुबारा  लोड करें और सत्यापित करें कि वांछित संस्करण का उपयोग किया जा रहा है:
+```bash
+echo $BASH_VERSION $SHELL
+```
+
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:34]:
```suggestion

Homebrew आमतौर पर इसे `/usr/local/bin/bash` पर स्थापित करता है।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+### इनस्टॉल बैश कम्पलीशन
+
+{{<ध्यान दे >}}
+जैसा कि उल्लेख किया गया है, ये निर्देश मानते हैं कि आप बैश 4.1+ का उपयोग करते हैं, जिसका अर्थ है कि आप बैश-पूर्णता v2 इनस्टॉल  करेंगे (बैश 3.2 और बैश-पूर्णता v1 के विपरीत, जिस स्थिति में kubectl  पूर्णता काम नहीं करेगी)।
+{{<ध्यान दे >}}
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:39]:
```suggestion
{{< note >}}
जैसा कि उल्लेख किया गया है, ये निर्देश मानते हैं कि आप Bash 4.1+ का उपयोग करते हैं, जिसका अर्थ है कि आप bash-completion v2 इनस्टॉल  करेंगे (Bash 3.2 और bash-completion v1 पर kubectl  पूर्णता काम नहीं करेगी)।
{{< /note >}}
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+आप `type_init_completion` से परीक्षण कर सकते हैं कि क्या आपके पास बैश कम्पलीशन v2 पहले से ही इनस्टॉल है | अगर नहीं, तो आप इसे Homebrew के साथ इनस्टॉल कर सकते हैं
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:41]:
```suggestion
आप `type_init_completion` से सत्यापित कर सकते हैं कि क्या आपके पास bash-completion v2 पहले से इनस्टॉल है। यदि नहीं, तो आप इसे Homebrew से इनस्टॉल कर सकते हैं
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+जैसा कि इस कमांड के आउटपुट में बताया गया है, अपनी `~/.bash_profile` फ़ाइल में निम्नलिखित जोड़ें:
+
+```bash
+export BASH_COMPLETION_COMPAT_DIR="/usr/local/etc/bash_completion.d"
+
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
+```
+  
+अपने शेल को दुबारा लोड करें और सत्यापित करें कि बैश कम्पलीशन v2 `type_init_completion` . के साथ सही ढंग से इनस्टॉल है
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:55]:
```suggestion
अपने शेल को पुनः लोड करें और `type_init_completion` से सत्यापित करें कि bash-completion  v2 सही ढंग से इनस्टॉल है।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+### इनेबल kubectl ऑटोकम्पलीशन
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:57]:
```suggestion
### kubectl ऑटोकम्पलीशन सक्षम करें
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+
+अब आपको यह सुनिश्चित करना होगा कि आपके सभी शेल सत्रों में kubectl कम्पलीशन स्क्रिप्ट प्राप्त हो जाए। इसे हासिल करने के कई तरीके हैं:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:59]:
```suggestion
अब आपको यह सुनिश्चित करने की आवश्यकता है कि kubectl समापन स्क्रिप्ट आपके सभी शेल सत्रों (sourced) में प्राप्त हो जाए। इसे हासिल करने के कई तरीके हैं:
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+
+- अपने कम्पलीशन स्क्रिप्ट को  `~/.bash_profile`  में सोर्स करें  
+```bash
+echo 'source <(kubectl completion bash)' >>~/.bash_profile
+```
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:64]:
```suggestion
- अपने कम्पलीशन स्क्रिप्ट को  `~/.bash_profile`  में सोर्स करें:

    ```bash
    echo 'source <(kubectl completion bash)' >>~/.bash_profile
    ```
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 16]
```diff
@@ -0,0 +1,84 @@
...
+
+- कम्पलीशन स्क्रिप्ट को `/usr/local/etc/bash_completion.d` डायरेक्टरी  में जोड़ें
+```bash
+kubectl completion bash >/usr/local/etc/bash_completion.d/kubectl
+```
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:69]:
```suggestion
- कम्पलीशन स्क्रिप्ट को `/usr/local/etc/bash_completion.d` डायरेक्टरी में जोड़ें:

    ```bash
    kubectl completion bash >/usr/local/etc/bash_completion.d/kubectl
    ```
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 17]
```diff
@@ -0,0 +1,84 @@
...
+
+- यदि आपके पास kubectl के लिए एक उपनाम है, तो आप उस उपनाम के साथ काम करने के लिए शेल पूर्णता को बढ़ा सकते हैं:
+```bash
+echo 'alias k=kubectl' >>~/.bash_profile
+
+echo 'complete -F __start_kubectl k' >>~/.bash_profile
+```
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:76]:
```suggestion
- यदि आपके पास kubectl के लिए एक नाम (alias) है, तो आप उस उपनाम के साथ काम करने के लिए शेल कम्पलीशन को बढ़ा सकते हैं:

    ```bash
    echo 'alias k=kubectl' >>~/.bash_profile
    echo 'complete -F __start_kubectl k' >>~/.bash_profile
    ```
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+
+- यदि आपने होमब्रे के साथ kubectl इनस्टॉल किया है (जैसा कि यहां बताया गया है), तो kubectl पहले से ही कम्पलीशन स्क्रिप्ट `/usr/local/etc/bash_completion.d/kubectl` में होनी चाहिए। ऐसे में आपको कुछ भी करने की जरूरत नहीं है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:78]:
```suggestion
- यदि आपने Homebrew के साथ kubectl इनस्टॉल किया है (जैसा कि यहां बताया गया है), तो kubectl कम्पलीशन स्क्रिप्ट पहले से ही `/usr/local/etc/bash_completion.d/kubectl` में होनी चाहिए। ऐसे में आपको कुछ भी करने की जरूरत नहीं है।
```

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+
+{{<ध्यान दे >}}
+बैश कम्पलीशन का homebrew से इनस्टॉल होना, सारे फाइल्स को `BASH_COMPLETION_COMPAT_DIR` डायरेक्टरी में सोर्स कर देता है | 
+इसिलिए बाद के दो तरीके काम करते है |
+{{<ध्यान दे >}}
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:83]:
```suggestion
{{< note >}}
bash-completion Homebrew से इनस्टॉल होने पर, सारे फाइल्स को `BASH_COMPLETION_COMPAT_DIR` डायरेक्टरी में सोर्स कर देता है। इसलिए आखरी दो तरीके काम करते हैं।
{{< /note >}}
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you, here's my review. Please go through all the suggested changes and commit if you agree with them. Feel free to ask if you have any doubt

File: content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+brew install bash
+```
+
+अपने शेल को पुनः लोड करें और सत्यापित करें कि इच्छित संस्करण का उपयोग किया जा रहा है:
+
+```bash
+echo $BASH_VERSION $SHELL
+```
+
+Homebrew आमतौर पर इसे `/usr/local/bin/bash` पर स्थापित करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-mac.md:37]:
```suggestion
Homebrew आमतौर पर इसे `/usr/local/bin/bash` पर इनस्टॉल करता है।

```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
Thank you @kranurag7 :)

**Comment by vedant-kakde** [comment]
Thanks @kranurag7

**Comment by kranurag7** [comment]
Thanks @anubha-v-ardhan & @vedant-kakde, I look forward to contribute more to the community.

**Pull request merged**